Source/Core/VideoCommon/VertexManagerBase.cpp

```diff
@@ -299,13 +302,16 @@ void VertexManager::CalculateZSlope(NativeVertexFormat* format)
 	// is enabled in the following flush.
 	for (unsigned int i = 0; i < 3; ++i)
 	{
-		u8* vtx_ptr = s_pCurBufferPointer - vert_decl.stride * (3 - i);
+		u8* vtx_ptr = s_pCurBufferPointer - vert_decl.stride * (3 - i) + vert_decl.position.offset;
```

Comment:
I'm sorry, the vert_decl.position.offset is already observed in the next line by posOff.